profile-diff/constants.js

```diff
@@ -0,0 +1,22 @@
+const YEARS_OF_EXPERIENCE = 'yoe';
+
+const fieldDisplayName = {
+  first_name: 'First name',
+  last_name: 'Last name',
+  designation: 'Role',
+  company: 'Company',
+  yoe: 'Years of experience',
+  email: 'Email',
+  phone: 'Phone no.',
+  linkedin_id: 'Linkedin',
+  github_id: 'Github',
+  twitter_id: 'Twitter',
+  instagram_id: 'Instagram',
+  website: 'Website',
```

Comment:
Nit: Keys can be `capitalized`

Reply:
Can't capitalize them as I am matching them with user data.